src/backend/InvenTree/InvenTree/api.py

```diff
@@ -546,10 +546,18 @@ class MetadataView(RetrieveUpdateAPI):
 
     MODEL_REF = 'model'
 
+    def __init__(self, *args, **kwargs):
+        """Initialize the MetadataView."""
+        super().__init__(*args, **kwargs)
+
```

Comment:
Question: Is this required? Seems to do nothing on a skim

Reply:
Thanks, was a stub from an experiment that is no longer needed - I will fix it.